zastosowania-komputerow-w-elektrotechnice.csl

```diff
@@ -0,0 +1,321 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only">
+  <info>
+    <title>Zastosowania Komputerów w Elektrotechnice</title>
+    <title-short>ZKwE</title-short>
+    <id>http://www.zotero.org/styles/zastosowania-komputerow-w-elektrotechnice</id>
+    <link href="http://www.zotero.org/styles/zastosowania-komputerow-w-elektrotechnice" rel="self"/>
+    <link href="http://www.zotero.org/styles/ieee" rel="template"/>
+    <link href="http://www.iee.put.poznan.pl/caiee/redakcja.doc" rel="documentation"/>
```

Comment:
that link should probably be http://www.iee.put.poznan.pl/zkwe/download/guide.doc
this one goes to a 404 for me.